web-ui/src/App.css

```diff
@@ -9,9 +9,9 @@
 }
 
 @media print {
-    :root {
```

Comment:
Lots of formatting changes follow!
I ran the app locally and didn't see any issues.
I also tried to verify that all the UI tests still pass. 7 failed. I'm investigating those now.

Reply:
I had to update some snapshots related to whitespace. Now all the UI tests pass.

Reply:
🎉